### DISCUSSION TITLE
Possibility to hide infobar at bottom of screen?

### DISCUSSION THREAD

**Discussion by aluminum1** [ahrm/sioyek#656] [General]
Hi - great work on an amazing product. I am wondering if it is possible to hide the black info bar at the bottom of the screen ("Page 3/3 etc). I am thinking of an option to make it only appear when you're typing in a command.

**Comment by ahrm** [comment]
You can turn in off using `toggle_statusabar`.

**Reply by aluminum1** [reply]
Many thanks.
